src/containers/landing/Landing.tsx

```diff
@@ -1,31 +1,66 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
 import MapPage from '../../components/map-page-components/map-page/MapPage';
+import MobileMapPage from '../../components/map-page-components/mobile-map-page/MobileMapPage';
+import MobileLandingBar from '../../components/map-page-components/mobile-landing-bar/MobileLandingBar';
 import LandingTreeStats from '../../components/landing-tree-stats/LandingTreeStats';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
 
 const Landing: React.FC = () => {
-  return (
-    <>
-      <Helmet>
-        <title>Landing</title>
-        <meta
-          name="description"
-          content="The first page someone sees if they are not logged in, contains a read only map of Boston neighborhoods and some information about the tree counts of Speak for the Trees"
-        />
-      </Helmet>
+  const { windowType } = useWindowDimensions();
 
-      <MapPage
-        sidebarHeader="Boston's Street Trees"
-        sidebarDescription="Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer cronut pok pok gentrify flannel salvia deep v pork belly pitchfork. Swag fashion axe fam. Occupy biodiesel jean shorts affogato PBR&B freegan bushwick vegan four loko pickled."
-      >
-        <LandingTreeStats
-          moneySaved={100000}
-          rainWater={100000}
-          carbonEmissions={31}
-        />
-      </MapPage>
-    </>
-  );
+  const statMoneySaved = 100000;
+  const statRainWater = 100000;
+  const statCarbonEmissions = 31;
+
+  switch (windowType) {
+    case WindowTypes.Mobile:
+      return (
+        <>
+          <Helmet>
+            <title>Landing</title>
```

Comment:
Let's rename the title, this sets the browser tab content

Reply:
`Speak for the Trees`?

Reply:
Perfect